changelog.txt

```diff
@@ -18,6 +18,7 @@ that repo.
 - `devel/luacov`: generate code test coverage reports for script development. define the ``DFHACK_ENABLE_LUACOV=1`` environment variable to start gathering coverage metrics.
 - `gui/mass-remove`: mass removal/suspension tool for buildings and constructions
 - `reveal-hidden-sites`: exposes all undiscovered sites
+- `set-timeskip-duration`: changes the duration of the `Updating World` process preceding the start of a new game, enabling you to jump in earlier or later than usual
```

Comment:
Got it, thanks. I'm not familiar with RST and the docs build error wasn't particularly helpful at identifying the cause of the problem. Will edit accordingly.

Reply:
Thanks - the changelog is particularly confusing because it gets included in both `news.rst` and `news-dev.rst`, and due to being included, line numbers don't match up either.